babi/file.py

```diff
@@ -465,6 +465,14 @@ def page_down(self, margin: Margin) -> None:
             self.buf.y = self.buf.file_y = pos
         self.buf.x = 0
 
+    @action
+    def paragraph_down(self, margin: Margin) -> None:
```

Comment:
these should be named after the key press like the other movement things (so this would be `alt_down`